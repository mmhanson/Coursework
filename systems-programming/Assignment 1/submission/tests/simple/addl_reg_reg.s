main:
	movl	$1, %ecx
	addl	%ecx, %ecx
	printr	%ecx
	addl	%ecx, %ebx
	printr	%ebx
