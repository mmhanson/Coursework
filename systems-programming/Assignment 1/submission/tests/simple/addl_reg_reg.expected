2 (0x2)
2 (0x2)
